content/md/what-is/understand-data-quality.md

```diff
@@ -44,31 +34,43 @@ Spell check is usable with localizable text and text area attributes.
 If you use a single locale, then spell check is usable on text and text area attributes.
 :::
 
+## What does the quality score mean?
+The **quality score** calculates the achievement
+
+- **A** stands for **"Excellent": 90 to 100%** of the data quality criteria are fulfilled. You may still have little improvements to make to reach perfection but you have done a tremendous job already.
+- **B** stands for **"Good": 80 to 89%** of the data quality criteria are fulfilled. This already is a good achievement, you can definitely go to the next level by following the PIM's recommendations and reach a first-class catalog.
+- **C** stands for **"Average": 70 to 79%** of the data quality criteria are fulfilled. Follow the PIM's recommendations whether in the dashboard or in the data quality insights panel of the product edit form to improve your data quality.
+- **D** stands for **"Below average": 60 to 69%** of the data quality criteria are fulfilled. Don't give up, we are here to help: check the **key indicators** to see where you should focus your efforts to begin with.
+- **E** stands for **"To improve": 0 to 59%** of the data quality criteria are fulfilled. You may be at the beginning of a project with a lot of enrichment to do or have lots of new products to enrich, or you may have unrelevant attribute groups activated such as ERP data downgrading your quality score.
+
 ::: tips
-From the `data quality insights` panel, by clicking on the "Display all improvable attributes", you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
+By disabling specific attribute groups in Akeneo settings, you can focus your insights and progress on the most relevant product data for your end-customer needs. Therefore, you will get insights on what’s truly important to improve your Product Experience.
```

Comment:
It's better to link to the article explaining how to disable attribute groups, instead of explaining here how to do.
```suggestion
By [disabling specific attribute group](manage-data-quality.html#data-quality-insights-activation-and-deactivation), you can focus your insights and progress on the most relevant product data for your end-customer needs. Therefore, you will get insights on what’s truly important to improve your product experience.
```